build.rs

```diff
@@ -53,7 +53,9 @@ fn main() {
                .define("USE_FIELD_INV_BUILTIN", Some("1"))
                .define("USE_SCALAR_INV_BUILTIN", Some("1"))
                .define("USE_ENDOMORPHISM", Some("1"))
-               .define("ENABLE_MODULE_ECDH", Some("1"));
+               .define("ENABLE_MODULE_ECDH", Some("1"))
+               .define("USE_EXTERNAL_DEFAULT_CALLBACKS", Some("1"))
+               .define("ECMULT_WINDOW_SIZE", Some("15")); // This is the default in the configure file (`auto`)
```

Comment:
ECmult window size needs to be configurable, 15 is definitely too large for smaller platforms
(though I don't know if rust has any way to pass values instead of feature bits?)

Reply:
I don't think there is a way in rust to pass variables as compile time arguments. but i'll look into it, maybe there's some hack we can use (maybe environment variable?)